assets/app/scripts/services/metrics.js

```diff
@@ -113,13 +123,19 @@ angular.module("openshiftConsole")
         return getMetricsURL().then(function(metricsURL) {
           var reqURL,
               template = metricsURL + templateByMetric[config.metric],
+              containerName = config.containerName,
               buckets = 60,
               start,
               end = config.end || Date.now();
 
+          // network metrics are not available at container level (only at pod and node level)
```

Comment:
Here's one idea. Subheadings are displayed here but they only show if the pod has more than one container. 

![screenshot from 2016-04-08 16-57-40](https://cloud.githubusercontent.com/assets/158611/14395939/1f4b2332-fdab-11e5-9362-0da67c8c9c43.png)